### ISSUE TITLE
Exception when iDisplayLength=-1 (all)

### ISSUE THREAD

**Issue by javiersanp** [pivotal-energy-solutions/django-datatable-view#119]
In version '0.9.0', when you make a request to DatatableView with iDisplayLenght set to -1 (all registers), the following exception is raised:

UnboundLocalError at / local variable 'object_list' referenced before assignment
Traceback:
File "/home/javier/.virtualenvs/ssbb_dev/lib/python3.4/site-packages/datatableview/datatables.py" in _get_current_page
1.         return object_list

The code of _get_current_page method of Datatable class have an if statement that leaves the variable object_list unassigned when self.config['page_length'] == -1

You can fix it adding a line with object_list = self._records just before the if.

Best regards.

**Comment by javiersanp** [comment]
The code after the change propossed is this:

```
    def _get_current_page(self):
        object_list = self._records
        if self.config['page_length'] != -1:
            i_begin = self.config['start_offset']
            i_end = self.config['start_offset'] + self.config['page_length']
            object_list = self._records[i_begin:i_end]
        return object_list
```

**Comment by paolodina** [comment]
It works for me.

**Comment by tiliv** [comment]
Fixed as if merge of #171 !
